tests/unit/database_tests.py

```diff
@@ -442,6 +442,17 @@ def test_all_docs_get(self):
         self.assertEqual(data['rows'][1]['key'], 'julia011')
         self.assertEqual(data['rows'][2]['key'], 'julia012')
 
+    def test_all_docs_get_with_long_type(self):
+        """
+        Test the all_docs GET request functionality
+        """
+        self.populate_db_with_documents()
+        data = self.db.all_docs(limit=10000000000, skip=10)
```

Comment:
@rhyshort @smithsz Test cases won't pass with `L` suffix since all integers in Python 3 are long integers.  I've added a `LONG_NUMBER` variable that's bigger than `sys.maxint` in 79e1e34.